caffe2/core hipify

xw285cornell opened the pull request:
Summary: Small edits to caffe2/core hipify to make it compile in fbcode.
Differential Revision: D12883472

bddppq commented:
https://ci.pytorch.org/jenkins/job/caffe2-builds/job/py2-clang7-rocmdeb-ubuntu16.04-trigger-test/4597/
https://ci.pytorch.org/jenkins/job/pytorch-builds/job/py2-clang7-rocmdeb-ubuntu16.04-trigger/4591/

bddppq reviewed (approved):
LG

xw285cornell committed:
caffe2/core hipify (#13457)

Summary:
Small edits to caffe2/core hipify to make it compile in fbcode.
Pull Request resolved: https://github.com/pytorch/pytorch/pull/13457

Reviewed By: bddppq

Differential Revision: D12883472

Pulled By: bddppq

fbshipit-source-id: c0f4d81e9cf99f7e4020fd7502ce9dcecc9ea047
